.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ on:
     push:
         branches: [master, develop]
     pull_request:
-        branches: [master, develop, refactor/react]
+        branches: [master, develop, refactor/react, refactor/raect/mvp]
```

Comment:
Just to mention this for later, we should clean-up this line after all merges

Reply:
Cleansed in 6baf7b1102f5b5936f4fbdd4c49706ab8f50a3bb